src/wallets/IdentityWallet.ts

```diff
@@ -480,3 +484,22 @@ export function calculateIdentityAddress(
       .slice(2 + 2 * 12)
   return toChecksumAddress(address)
 }
+
+/**
+ * Generates a random nonce to use for meta transactions.
+ * The nonce fits into the range of ]2^255, 2^256[.
+ * This is an alternative to the up counting nonce (]0, 2^255[) without the need
+ * to query a [[TLProvider]].
+ */
+export function getRandomNonce(): string {
+  const random = BigNumber.random()
+  const nonce = random.multipliedBy(RANDOM_NONCE_RANGE).plus(MIN_RANDOM_NONCE)
+  const coefficient = nonce.c
+  const exponent = nonce.e
+  const nonceAsString = coefficient.reduce(
+    (nonceString, coefficientPart) =>
+      nonceString.concat(coefficientPart.toString()),
+    ''
+  )
+  return nonceAsString.substr(0, exponent + 1)
```

Comment:
Why can you not just do `nonce.toString()`?

Reply:
Due to [the docs](https://mikemcl.github.io/bignumber.js/#exponential-at) it allows to configure the `EXPONENTIAL_AT` value for the `toString()` function only up to `1e+9`, which is not sufficient for us.

Reply:
As we discussed in person I used the configuration options wrongly during my
experiments so that is whole thing can be just simplified. Will apply that.